src/CircleCiOidcProvider.ts

```diff
@@ -16,47 +22,38 @@ export interface CircleCiOidcProviderProps {
    * If you do need to generate this thumbprint, you can follow the instructions here:
    * https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html
    */
-  readonly circleCiOidcThumbprints?: string[];
+  readonly thumbprints?: string[];
 }
 
 /**
  * This construct creates a CircleCI ODIC provider to allow AWS access from CircleCI jobs. You'll need to instantiate
  * this construct once per AWS account you want to use CircleCI OIDC with.
  *
+ * You can import a existing provider using `CircleCiOidcProvider.fromOrganizationId`.
+ *
  * To create a role that can be assumed by CircleCI jobs, use the `CircleCiOidcRole` construct.
  */
-export class CircleCiOidcProvider extends Construct {
```

Comment:
Instead of having a wrapping `Construct`, this now just "is a" `CfnOIDCProvider` for simplicity.